src/collective/fingerpointing/subscribers/registry_logger.py

```diff
@@ -23,6 +23,6 @@ def registry_logger(event):
 
         if IRecordModifiedEvent.providedBy(event):
             action = 'modify'
-            extras = 'object={0} value={1}'.format(event.record, event.record.value)
+            extras = u'object={0} value={1}'.format(event.record, event.record.value)
```

Comment:
Sorry, I didn't see @jianaijun comment, I'm looking how ``Plone.site_title`` works in Plone 5 now.